scripts/delete.rb

```diff
@@ -0,0 +1 @@
+Person.find_by(title:'Deceased').delete_all
```

Comment:
`.find_by` returns a Person object which does not have the `.delete_all` method. Try using `.destroy` instead